### ISSUE TITLE
Macular location

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11557]
Preferred term label:
Macular location

Synonyms

    Inside macular

    Intramacular location

    Macular region involvement

Definition (free text, please give PubMed ID)
An anatomical spatial pattern or localization restricted to or primarily involving the macula lutea of the retina. This modifier is used to specify that a retinal phenotype (e.g., flecks, edema, or hemorrhage) is centrally located within the anatomical macula. PMID:27746098

Parent term (use hpo.jax.org/app)
The anatomical localization of the specified phenotypic abnormality (HP:0012830)

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)

    Stargardt disease (OMIM:248200)

    Best vitelliform macular dystrophy (OMIM:153700)

Your nano-attribution (ORCID)
0000-0001-9042-0508
